Google.PowerShell/Common/MeasurementProtocolService.cs

```diff
@@ -61,6 +63,9 @@ public static void SetClientId(string clientId)
             ClientId = clientId.Trim();
         }
 
+        // Removed this part of the code for CoreCLR for now, Chris is working on a PR
+        // that will remove this file.
+#if !CORECLR
```

Comment:
Rather than ifdeffing out the method entirely AND its callsite, why don't we just stub it out? The result would be much easier to read IMHO. e.g.

````
public static HttpWebRequest GenerateRequest(string category, string action, string label, int? value = null) {
    #if CORECLR
    // TODO(quoct): Wire telemetry reporting into PowerShell for non-Windows platforms.
    // This requires finding an alternative to current use of System.Web.
    return;
    #else
    ...
    #endif
}
````

We might need a `    category = action = label = null;  // Prevent warnings.` in case the compiler whines.